packages/api/amplify_api/lib/src/amplify_api_config.dart

```diff
@@ -29,7 +29,8 @@ class EndpointConfig with AWSEquatable<EndpointConfig> {
 
   /// Gets the host with environment path prefix from Amplify config and combines
   /// with [path] and [queryParameters] to return a full [Uri].
-  Uri getUri(String path, Map<String, dynamic>? queryParameters) {
+  Uri getUri(String? path, Map<String, dynamic>? queryParameters) {
```

Comment:
request: Would it be possible to add unit test coverage for this class/utility here? I know it was not created in this PR but would be good to test the logic here regarding how it gets URIs from config, especially when inputs are null. Arguably this code could be simplified but would rather see a test, first.